_data/versions.json

```diff
@@ -1,6 +1,7 @@
 {
-  "current": "1.3",
+  "current": "2.0",
   "all": [
+    "2.0",
```

Comment:
Didn't we just remove this in another PR? Or is this something else?

Reply:
that for for 1.3

Reply:
This adds it back in. For some additional context, we've changed how versioning works for 2.0-rc1. We now have latest and current parameters. Furthermore, we'll now use branches for each dropdown version, allowing us to push into main earlier when developing new documentation.

This adds the 2.0 version back in, which will act as main until the next release. And 1.3.x changes will be merged into the 1.3 branch.